### PR TITLE
[ELY-1500] FactoryCredentialSource acquire support fix

### PR DESCRIPTION
**Pull request by hkalina**
https://issues.jboss.org/browse/ELY-1500

### CODE REVIEW

File: src/main/java/org/wildfly/security/credential/source/FactoryCredentialSource.java [hunk 1]
```diff
@@ -1,3 +1,21 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018 Red Hat, Inc., and individual contributors
```
Comment (darranl) [inline src/main/java/org/wildfly/security/credential/source/FactoryCredentialSource.java]: I think this probably should be 2017 as that is when this class was written.

**Comment by nekdozjam** [comment]
This is how it was meant to be, sorry for the mistake.

File: src/main/java/org/wildfly/security/credential/source/FactoryCredentialSource.java (hunk 1 above)
Comment (hkalina) [reply src/main/java/org/wildfly/security/credential/source/FactoryCredentialSource.java]: fixed (sorry, no idea why Idea shows me "18/01/18" in blame :D)

**Comment by hkalina** [comment]
@darranl as this has +2 already, is there some reason to not merge?

**Comment by darranl** [comment]
No I think I was waiting for CI to finish and moved onto something else, this is fine to go in.

**Pull request merged**